Rebased due to resyncing fork with main, although it appears the build is being aborted due to the changes made to the test match regex

```
Error: Files were changed during build (see build log). If this was triggered from a fork, you will need to update your branch.
diff --git a/package.json b/package.json
index 827d3df..040[9](https://github.com/cdklabs/cdk-pipelines-github/actions/runs/8563142043/job/23506037649#step:8:10)69e 100644
--- a/package.json
+++ b/package.json
@@ -96,8 +96,7 @@
   "jest": {
     "testMatch": [
       "<rootDir>/src/**/__tests__/**/*.ts?(x)",
-      "<rootDir>/test/**/*(*.)@(spec|test).ts?(x)",
-      "<rootDir>/src/**/*(*.)@(spec|test).ts?(x)"
+      "<rootDir>/(test|src)/**/*(*.)@(spec|test).ts?(x)"
     ],
     "clearMocks": true,
     "collectCoverage": true,
Error: Process completed with exit code 1.
```

--- a/.projenrc.ts
+++ b/.projenrc.ts
@@ -22,6 +22,12 @@ const project = new CdklabsConstructLibrary({
   ],
   peerDeps: ['aws-cdk-lib'],
   jestOptions: {
+    jestConfig: {
+      testMatch: [
+        '<rootDir>/test/**/*(*.)@(spec|test).ts?(x)',
+        '<rootDir>/src/**/*(*.)@(spec|test).ts?(x)',
+      ],
+    },
     updateSnapshot: UpdateSnapshot.NEVER,
   },
 

--- a/package.json
+++ b/package.json
@@ -95,9 +95,10 @@
   "version": "0.0.0",
   "jest": {
     "testMatch": [
-      "<rootDir>/src/**/__tests__/**/*.ts?(x)",
       "<rootDir>/test/**/*(*.)@(spec|test).ts?(x)",
-      "<rootDir>/src/**/*(*.)@(spec|test).ts?(x)"
+      "<rootDir>/src/**/*(*.)@(spec|test).ts?(x)",
+      "<rootDir>/src/**/__tests__/**/*.ts?(x)",
+      "<rootDir>/(test|src)/**/*(*.)@(spec|test).ts?(x)"
     ],
     "clearMocks": true,
     "collectCoverage": true,

